custom/opm/reports.py

```diff
@@ -1112,7 +1116,7 @@ def rows():
                 for method, __, __, denom in self.model.method_map:
                     value = getattr(awc, method)
                     row.append(value)
-                    if denom != 'no_denom':
+                    if denom != 'no_denom' and denom != 'one':
```

Comment:
current `method_map` convention doesn't allow for percentage on Totals only, so I used this non-neat way. Let me know if you see any probs with this @esoergel